templates/inbox/index.html

```diff
@@ -27,96 +27,80 @@
     <script>
         inboxitems.reverse();
         for(let inboxitem of inboxitems){
-            console
             author=inboxitem.author;
             if (!author && inboxitem.actor){
                 author = inboxitem.actor
             }
             var displayName;
-            url = '{{host}}'+'/author/'+author.id;
-            fetch(url,{
-                method: 'get',
-                headers: {
-                    'Content-Type' : 'application/json',
-                    'X-CSRFToken' : "{{ csrf_token }}"
-            },})
-            .then(function(authorResponse){
-                return authorResponse.json();
-            })
-            .then(function(authorData){
-                displayName = authorData.displayName;
-                console.log(authorData);
-            })
-            .then(function(){
-                h1 = document.createElement("h1");
-                h1.textContent = "Type: " + inboxitem.type;
+
+            h1 = document.createElement("h1");
+            h1.textContent = "Type: " + inboxitem.type;
+            li = document.createElement("li");
+            a = document.createElement("a");
+            list = document.getElementById("inboxitems")
+            li.textContent="By: " ;
+            a.textContent = displayName;
```

Comment:
you removed the part where you are retrieving displayName so it renders an empty tag